pycode/memilio-epidata/memilio/epidata/getDataIntoPandasDataFrame.py

```diff
@@ -296,28 +305,31 @@ def write_dataframe(df, directory, file_prefix, file_type):
     - json
     - json_timeasstring [Default]
     - hdf5
+    - txt
     The file_type defines the file format and thus also the file ending.
-    The file format can be json or hdf5.
+    The file format can be json, hdf5 or txt.
     For this option the column Date is converted from datetime to string.
 
     @param df pandas dataframe (pandas DataFrame)
     @param directory directory where to safe (string)
     @param file_prefix filename without ending (string)
     @param file_type defines ending (string)
+    @param param_dict defines parameters for to_csv/txt(dictionary)
 
     """
 
     outForm = {'json': [".json", {"orient": "records"}],
                'json_timeasstring': [".json", {"orient": "records"}],
-               'hdf5': [".h5", {"key": "data"}]}
+               'hdf5': [".h5", {"key": "data"}],
+               'txt': [".txt", param_dict]}
 
     try:
         outFormEnd = outForm[file_type][0]
         outFormSpec = outForm[file_type][1]
     except KeyError:
         raise ValueError(
             "Error: The file format: " + file_type +
-            " does not exist. Use json, json_timeasstring or hdf5.")
+            " does not exist. Use json, json_timeasstring, hdf5 or txt.")
```

Comment:
The change leads to an error message in the unit tests. Please adjust line 532 "test_epidata_getDataIntoPandasDataFrame.py".